examples/era5_cogs.md

```diff
@@ -0,0 +1,243 @@
+---
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.16.3
+  kernelspec:
+    display_name: venv
+    language: python
+    name: python3
+---
+
+# Exploring [ERA5 Reanalysis](https://cds-beta.climate.copernicus.eu/datasets/reanalysis-era5-single-levels-monthly-means?tab=overview) COGs
+
+This notebook takes a closer look at our team's store of ERA5 reanalysis outputs, stored as COGs in Azure blob storage. This data is downloaded originally from the [CDS Beta API](https://cds-beta.climate.copernicus.eu). We'll demonstrate a basic analysis where we calculate the average accumulated precipitation for the month of August across admin 1 regions in Haiti.
+
+```python
+%load_ext jupyter_black
+%load_ext autoreload
+%autoreload 2
+```
+
+```python
+import xarray as xr
+import pandas as pd
+import tqdm
+import os
+import re
+from dotenv import load_dotenv
+from azure.storage.blob import ContainerClient
+import rioxarray as rxr
+import zipfile
+import io
+import geopandas as gpd
+import matplotlib.pyplot as plt
+
+load_dotenv()
+
+PROD_BLOB_SAS = os.getenv("DSCI_AZ_SAS_PROD")
+PROD_BLOB_NAME = "imb0chd0prod"
```

Comment:
Maybe this needs to be changed in SEAS5 as well but since we have this in our policies it would be nice to pick up this config from our env variables. Same for RASTER_CONTAINER_NAME and DEV_BLOB_NAME.

Reply:
Good catch!